src/Unity.Mathematics/Geometry/MinMaxAABB.cs

```diff
@@ -1,49 +1,331 @@
 using System;
+using System.Runtime.CompilerServices;
 using static Unity.Mathematics.math;
 
 namespace Unity.Mathematics.Geometry
 {
+    /// <summary>
+    /// Axis aligned bounding box (AABB) stored in min/max form.
+    /// </summary>
+    /// <remarks>
+    /// Axis aligned bounding boxes (AABB) are boxes where each side is parallel with one of the Cartesian coordinate axes
+    /// X, Y, and Z. AABBs are useful for approximating the region an object (or collection of objects) occupy and quickly
+    /// testing whether or not that object (or collection of objects) is relevant. Because they are axis aligned, they
+    /// are very cheap to construct and perform overlap tests with them.
+    /// </remarks>
     [System.Serializable]
     internal struct MinMaxAABB : IEquatable<MinMaxAABB>
     {
+        /// <summary>
+        /// The most minimum point contained by the AABB.
+        /// </summary>
+        /// <remarks>
+        /// If <see cref="Min"/> is greater than <see cref="Max"/> then this AABB is invalid.
```

Comment:
There is no order defined on float3, so I guess this means when any component is greater?